.github/workflows/ci.yml

```diff
@@ -16,14 +16,28 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-      
+
+    - name: Build panacea-core
+      env:
+        OWNER_MNEMONIC: genuine key outside escape oval unhappy mansion cricket practice quarter purchase picnic layer bicycle stem soup column creek convince obey rather rice there alcohol
+        TO_MNEMONIC: margin river antenna nest drama combine crystal clarify route country genuine acoustic hold ocean image fox flame invite scrub chalk destroy resource item final
+        DOCKER_NAME: panacea-core-${{ github.sha }}
+        DOCKER_TAG: panacea-core:${{ github.sha }}
+      run: |
+        echo "OWNER_MNEMONIC=${OWNER_MNEMONIC}"
+        echo "TO_MNEMONIC=${TO_MNEMONIC}"
```

Comment:
I forgot to comment on this in the previous PR.
What do you think about rename this to `RECEIVER_MNEMONIC` or so?

Reply:
No problem. I'll try to fix it.